06_gpu_and_ml/comfyui/comfyapp.py

```diff
@@ -134,14 +129,20 @@ def ui():
 # To run a workflow as an API:
 # 1. Stand up a "headless" ComfyUI server in the background when the app starts.
 # 2. Define an `infer` method that takes in a workflow path and runs the workflow on the ComfyUI server.
-# 3. Stand up a web handler `api` with `web_endpoint`, so that we can run our workflow as a service and accept inputs from clients.
+# 3. Create a web handler `api` with `web_endpoint`, so that we can run our workflow as a service and accept inputs from clients.
 #
 # Group all these steps into a single Modal `cls` object, which we'll call `ComfyUI`.
 @app.cls(
     allow_concurrent_inputs=10,  # allow 10 concurrent API calls
     container_idle_timeout=300,  # 5 minute container keep alive after it processes an input; increasing this value is a great way to reduce ComfyUI cold start times
     gpu="L40S",
     volumes={"/cache": vol},
+    mounts=[  # mount workflow JSON that we want to serve to the container
+        modal.Mount.from_local_file(
+            Path(__file__).parent / "workflow_api.json",
+            remote_path="/root/workflow_api.json",
+        )
+    ],
```

Comment:
Sorry this was my bad, mounts on classes / functions is about to be deprecated (I think there's already a warning about it or about to be) so the previous implementation was correct with `add_local_file` on the Image. I was just suggesting to put it as the very last step of the image build so we don't need the `copy=True` and subsequent steps aren't invalidated whenever the workflow file changes